CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 ## New Features
 ## Improvements
 ## Fixes
+- Fix npm 7 installation issue, migrate GraphQL playground to GraphIQL  (Daniel Blyth) [#1807](https://github.com/parse-community/parse-dashboard/pull/1807)
```

Comment:
Can you remove this change? the changelog entry will be created automatically using the pull request title